src/schema/alter-table-builder.ts

```diff
@@ -239,6 +242,24 @@ export class AlterTableBuilder implements ColumnAlteringInterface {
     })
   }
 
+  addIndex(indexName: string): AlterTableAddIndexBuilder {
```

Comment:
missing jsdocs here. you know the usual description + example.

Reply:
Done ✅ https://github.com/kysely-org/kysely/pull/720/commits/ea88a775c627880032ac46b7352954ff54306cd3